Furball.Vixie/Graphics/Backends/OpenGL20/OpenGL20Backend.cs

```diff
@@ -176,5 +179,122 @@ public class OpenGL20Backend : GraphicsBackend, IGLBasedBackend {
+            this.gl.ShaderSource(shaderId, source);
+        }
+        public void CompileShader(uint shaderId) {
+            this.gl.CompileShader(shaderId);
+        }
+        public string GetShaderInfoLog(uint shaderId) => this.gl.GetShaderInfoLog(shaderId);
+        public void AttachShader(uint programId, uint shaderId) {
+            this.gl.AttachShader(programId, shaderId);
+        }
+        public void LinkProgram(uint programId) {
+            this.gl.LinkProgram(programId);
+        }
+        public void GetProgram(uint programId, ProgramPropertyARB linkStatus, out int i) {
+            this.gl.GetProgram(programId, (GLEnum)linkStatus, out i);
+        }
+        public void DeleteShader(uint shader) {
+            this.gl.DeleteShader(shader);
+        }
+        public string GetProgramInfoLog(uint programId) => this.gl.GetProgramInfoLog(programId);
+        public void UseProgram(uint programId) {
+            this.gl.UseProgram(programId);
+        }
+        public int GetUniformLocation(uint programId, string uniformName) => this.gl.GetUniformLocation(programId, uniformName);
+        public unsafe void UniformMatrix4(int getUniformLocation, uint i, bool b, float* f) {
+            this.gl.UniformMatrix4(getUniformLocation, i, b, f);
+        }
+        public void Uniform1(int getUniformLocation, float f) {
+            this.gl.Uniform1(getUniformLocation, f);
+        }
+        public void Uniform1(int getUniformLocation, int f) {
+            this.gl.Uniform1(getUniformLocation, f);
+        }
+        public void Uniform2(int getUniformLocation, float f, float f2) {
+            this.gl.Uniform2(getUniformLocation, f, f2);
+        }
+        public void Uniform2(int getUniformLocation, int f, int f2) {
+            this.gl.Uniform2(getUniformLocation, f, f2);
+        }
+        public void DeleteProgram(uint programId) {
+            this.gl.DeleteProgram(programId);
+        }
+        public void DeleteVertexArray(uint arrayId) {
+            this.gl.DeleteVertexArray(arrayId);
+        }
+        public uint GenVertexArray() => this.gl.GenVertexArray();
+        public void EnableVertexAttribArray(uint u) {
+            this.gl.EnableVertexAttribArray(u);
+        }
+        public unsafe void VertexAttribPointer(uint u, int currentElementCount, Silk.NET.OpenGL.VertexAttribPointerType currentElementType, bool currentElementNormalized, uint getStride, void* offset) {
+            this.gl.VertexAttribPointer(u, currentElementCount, (GLEnum)currentElementType, currentElementNormalized, getStride, offset);
+        }
+        public unsafe void VertexAttribIPointer(uint u, int currentElementCount, Silk.NET.OpenGL.VertexAttribIType vertexAttribIType, uint getStride, void* offset) {
+            this.gl.VertexAttribIPointer(u, currentElementCount, (GLEnum)vertexAttribIType, getStride, offset);
+        }
+        public void BindVertexArray(uint arrayId) {
+            this.gl.BindVertexArray(arrayId);
+        }
```

Comment:
at the VERY least seperate each function with 1 new line, this is all crammed together so tightly so thats the VERY least you can do